src/maintainer/updating_pkgs.rst

```diff
@@ -222,8 +222,12 @@ Sometimes mistakes happen and a broken package ends up being uploaded to the con
 
 If the only issue is in the package metadata, we can directly patch it using
 the `repo data patches feedstock <https://github.com/conda-forge/conda-forge-repodata-patches-feedstock>`__.
-Please make a PR there to add a patch. In order to ensure future versions have the required changes, you also
-need to change the recipe to reflect the metadata changes.
+If this is the case, the following general guidelines should be followed:
+1. Update the feedstocks recipe to ensure future builds do not propagate the issue with a new build number.
+2. Please make a PR there to add a patch. The patch should specify as much has possible the versions
+   and times when the packages were generated. It may use the following information
+   - The current timestamp, you may generate it with ``python -c "import time; print(f'{time.time():.0f}000')"``.
+   - The problematic version and build numbers of the packages to affect.
```

Comment:
```suggestion
2. Please make a PR there to add a patch. The patch should specify as much has possible the versions and times when the packages were generated. It may use the following information
   
   - The current timestamp, you may generate it with ``python -c "import time; print(f'{time.time():.0f}000')"``.
   - The problematic version and build numbers of the packages to affect.
```